src/routes/auth.js

```diff
@@ -1,34 +1,84 @@
 import express from 'express';
-import { verify } from 'jsonwebtoken';
-import { secretkey } from '../config/keys';
+import jwt, { verify } from 'jsonwebtoken';
+import { secretkey, expTime, rememberTime } from '../config/keys';
 
 const router = express.Router();
 
-// post route to check validity of tokens, clients will hit this route.
-router.post('/', (req, res) => {
-    // extract token from cookie
-    const token = req.header('auth-token');
+function createJWTCookie(user, res, tokenName = 'token') {
+    const payload = {
```

Comment:
Move these functions to a new file called `utils.js` inside `utils` folder for now and export from there.

Reply:
Also use arrow style functions for consistency.
kind of like
```js
export const verifytoken = (req,token) => {
   // do whatever
}
```